app/src/main/java/com/yagni/view/EmployeeGUI.java

```diff
@@ -8,154 +8,186 @@
 import com.yagni.controller.*;
```

Comment:
delete whitespace on line 7. move import com.yagni up to line 7

Reply:
removed!